sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClient.java

```diff
@@ -1115,8 +1123,10 @@ Response<BinaryData> getAudioTranslationAsResponseObjectWithResponse(String depl
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<BinaryData> getAudioTranslationAsPlainTextWithResponse(String deploymentOrModelName,
+    Response<BinaryData> getAudioTranslationAsPlainTextWithResponse(String deploymentOrModelName,
```

Comment:
When manually changing the method, you should also remove `@Generated` annotation.

Reply:
it is automatically. Weidong fixes the issue in https://github.com/Azure/azure-rest-api-specs/commit/da72197d2ab5d7b0d2756976a3cf2927fc60adc4